### ISSUE TITLE
allowSchedulingOnControlPlanes missing from gen config

### ISSUE THREAD

**Issue by si458** [siderolabs/talos#7313]
### Description
when you generate a config with comments, `allowSchedulingOnControlPlanes` is missing completely from the comments
`.\talosctl-windows-amd64.exe gen config mycluster https://192.168.168.233:6443 --output-dir mycluster --kubernetes-version 1.26.4`

### Environment

- Talos version: [`talosctl version --nodes <problematic nodes>`] ANY
- Kubernetes version: [`kubectl version --short`] ANY
- Platform: ANY

**Comment by smira** [comment]
Yes, in fact many options are missing, as we can't put each and every option, but this one probably deserves to be there.

**Comment by si458** [comment]
@smira yes please add it! i was getting so confused as the online said its there but i couldnt find it anywhere in the config that was generated, i throught it might of been a hidden option maybe for advanced users
